server/halomod_app/__init__.py

```diff
@@ -148,6 +108,7 @@ def get_plot_types():
     #              <model_label>: {"xs": <array_of_xs>, "ys": <array_of_ys>},
     #              ...
     #           }}
+
     @app.route('/get_plot_data', methods=["POST"])
     def get_plot_data():
 
```

Comment:
Not super clear on why some endpoints were moved to blueprints and others weren't?
Same with conversion of comments to docstrings.

I get that it wasn't in the original scope of this user story but the inconsistency makes the server-side code a little harder to follow and navigate. Not really sure what the solution is except completing the abstraction or moving the abstraction to another US / branch.

Reply:
The goal was to test the waters. Seems like it did it's job. If it needed to be reverted, then it wouldn't be too hard, and if it needed to be completed, still wouldn't be too hard. See other comment below for a more thorough explanation.